src/test/unit.test.js

```diff
@@ -84,7 +84,7 @@ test('Test jobStatusText', (t) => {
 // jobPrefIsPublic function tests
 
 test('Test jobPrefIsPublic', (t) => {
-  t.equals(jobPrefIsPublic('private'), false, 'jobPrefIsPrivate(\'private\') returns correct result');
-  t.equals(jobPrefIsPublic('public'), true, 'jobPrefIsPrivate(\'public\') returns correct result');
+  t.equals(jobPrefIsPublic('private'), 'lock icon', 'jobPrefIsPrivate(\'private\') returns correct result');
```

Comment:
i think using something like visible / not_visible would be clearer here rather than lock / eye icon which seems based on css that might change.